

============
Publications
============

Import to Reference Manager
---------------------------

Download :download:`as a .CSV file <../downloads/grdi-amr_ref.csv>`.  

Download :download:`as a .RIS file <../downloads/grdi-amr_ref.ris>`.  

Download :download:`as a .BIB file <../downloads/grdi-amr_ref.bib>`.  


Unsorted
--------

Angers-Loustau, A., Petrillo, M., Bengtsson-Palme, J., Berendonk, T., Blais, B., Chan, K.-G., … Van den Eede, G. (2018). The challenges of designing a benchmark strategy for bioinformatics pipelines in the identification of antimicrobial resistance determinants using next generation sequencing technologies. F1000Research, 7, 459. `Full Text <https://doi.org/10.12688/f1000research.14509.1>`_

Banerjee, S. K., & Farber, J. M. (2018). Trend and Pattern of Antimicrobial Resistance in Molluscan Vibrio Species Sourced to Canadian Estuaries . Antimicrobial Agents and Chemotherapy, 62(10), 1–9. `Full Text <https://doi.org/10.1128/aac.00799-18>`_

Banerjee, S. K., Rutley, R., & Bussey, J. (2018). Diversity and Dynamics of the Canadian Coastal Vibrio Community: an Emerging Trend Detected in the Temperate Regions . Journal of Bacteriology, 200(15), 1–4. `Full Text <https://doi.org/10.1128/jb.00787-17>`_

Barker, D. O., Carrico, J. A., Kruczkiewicz, P., Palma, F., Rossi, M., & Taboada, E. N. (2018). Rapid Identification of Stable Clusters in Bacterial Populations Using the Adjusted Wallace Coefficient. BioRxiv, 299347. `Full Text <https://doi.org/10.1101/299347>`_

Bioteau, A., Huguet, K., Burrus, V., & Banerjee, S. (2018). Genome Sequence of a Canadian Vibrio parahaemolyticus Isolate with Unique Mobilizing Capacity. Genome Announcements, 6(24), 4–5. `Full Text <https://doi.org/10.1128/genomeA.00520-18>`_

Blais, B. W., Tapp, K., Dixon, M., & Carrillo, C. D. (2018). Genomically Informed Strain-Specific Recovery of Shiga Toxin–Producing Escherichia coli during Foodborne Illness Outbreak Investigations. Journal of Food Protection, 82(1), 39–44. `Full Text <https://doi.org/10.4315/0362-028x.jfp-18-340>`_

Brown, E. E. F., Cooper, A., Carrillo, C., & Blais, B. (2019). Selection of Multidrug-Resistant Bacteria in Medicated Animal Feeds. Frontiers in Microbiology, 10(March), 1–8. `Full Text <https://doi.org/10.3389/fmicb.2019.00456>`_

Cameron, A., Barbieri, R., Read, R., Church, D., Adator, E. H., Zaheer, R., & McAllister, T. A. (2019). Functional screening for triclosan resistance in a wastewater metagenome and isolates of Escherichia coli and Enterococcus spp. From a large Canadian healthcare region. PLoS ONE, 14(1), 1–14. `Full Text <https://doi.org/10.1371/journal.pone.0211144>`_

Cameron, A., Klima, C. L., Ha, R., Gruninger, R. J., Zaheer, R., & McAllister, T. A. (2018). A Novel aadA Aminoglycoside Resistance Gene in Bovine and Porcine Pathogens. MSphere, 3(1), 1–6. `Full Text <https://doi.org/10.1128/mSphere.00568-17>`_

Chekabab, S. M., Rehman, M. A., Yin, X., Carrillo, C., Mondor, M., & Diarra, M. S. (2019). Growth of Salmonella enterica Serovars Typhimurium and Enteritidis in Iron-Poor Media and in Meat: Role of Catecholate and Hydroxamate Siderophore Transporters. Journal of Food Protection, 82(4), 548–560. `Full Text <https://doi.org/10.4315/0362-028X.JFP-18-371>`_

Collineau L, Chapman B, Bao X, Sivapathasundaram B, Carson CA, Fazil A, Reid-Smith RJ, Smith BA. 2020. A farm-to-fork quantitative risk assessment model for Salmonella Heidelberg resistant to third-generation cephalosporins in broiler chickens in Canada. International Journal of Food Microbiology. Volume 330. `Full Text <https://doi.org/10.1016/j.ijfoodmicro.2020.108559>`_

Cooper, A. L., Low, A. J., Koziol, A. G., Thomas, M. C., Leclair, D., Tamber, S., Wong, A., Blais, B. W., & Carrillo, C. D. (2020). Whole Genome Sequence-Based Predictions of Serotype and Antimicrobial Resistance for Salmonella Isolates from Canadian Poultry. Frontiers in Microbiology, 11, 549. `Full Text <https://doi.org/10.3389/fmicb.2020.00549>`_

Islam, M. R., Lepp, D., Godfrey, D. V, Orban, S., Ross, K., Delaquis, P., & Diarra, M. S. (2019). Effects of wild blueberry ( Vaccinium angustifolium ) pomace feeding on gut microbiota and blood metabolites in free-range pastured broiler chickens . Poultry Science, 1–17. `Full Text <https://doi.org/10.3382/ps/pez062>`_

Klima, C. L., Holman, D. B., Ralston, B. J., Stanford, K., Zaheer, R., Alexander, T. W., & McAllister, T. A. (2019). Lower Respiratory Tract Microbiome and Resistome of Bovine Respiratory Disease Mortalities. Microbial Ecology. `Full Text <https://doi.org/10.1007/s00248-019-01361-3>`_

Lau, C. H., DeJong, E. N., Dussault, F., Carrillo, C., Stogios, P.J., & Topp, E. (2020). A penicillin-binding protein that can promote advanced-generation cephalosporin resistance and genome adaptation in the opportunistic pathogen Pseudomonas aeruginosa. International Journal of Antimicrobial Agents. 55(3), 105896. `Full Text <https://doi.org/10.1016/j.ijantimicag.2020.105896>`_

Le, K. K., Whiteside, M. D., Hopkins, J. E., Gannon, V. P. J., & Laing, C. R. (2018). Spfy: An integrated graph database for real-time prediction of bacterial phenotypes and downstream comparative analyses. Database, 2018(2018), 1–10. `Full Text <https://doi.org/10.1093/database/bay086>`_

Low, A. J., Koziol, A. G., Manninger, P. A., Blais, B. W., & Carrillo, C. D. 2019. ConFindr: Rapid detection of intraspecies and cross-species contamination in bacterial whole-genome sequence data. PeerJ. 7, e6995. `Full Text <http://doi.org/10.7717/peerj.6995>`_

MacKinnon MC, Sargeant JM, Pearl DL, Reid-Smith RJ, Carson CA, Parmley EJ, McEwen SA Evaluation of the health and healthcare system burden due to antimicrobial-resistant Escherichia coli infections in humans: a systematic review and meta-analysis. MacKinnon MC, Sargeant JM, Pearl DL, Reid-Smith RJ, Carson CA, Parmley EJ, McEwen SA. Antimicrob Resist Infect Control. 2020 Dec 10;9(1):200.

Maguire, F., Rehman, M. A., Carrillo, C., Diarra, M. S., & Beiko, R. G. (2019). Identification of primary antimicrobial resistance drivers in agricultural nontyphoidal Salmonella enterica serovars by using machine learning. MSystems, 4, e00211-19.  `Full Text <https://doi.org/10.1128/mSystems.00211-19>`_

McAllister, T. A., Wang, Y., Diarra, M. S., Alexander, T., & Stanford, K. (2018). Challenges of a one-health approach to the development of alternatives to antibiotics. Animal Frontiers, 8(2), 10–20. `Full Text <https://doi.org/10.1093/af/vfy002>`_

Murray, R., Tien, Y. C., Scott, A., & Topp, E. (2019). The impact of municipal sewage sludge stabilization processes on the abundance, field persistence, and transmission of antibiotic resistant bacteria and antibiotic resistance genes to vegetables at harvest. Science of the Total Environment, 651, 1680–1687. `Full Text <https://doi.org/10.1016/j.scitotenv.2018.10.030>`_

Palma, F., Manfreda, G., Silva, M., Parisi, A., Barker, D. O. R., Taboada, E. N., … Rossi, M. (2018). Genome-wide identification of geographical segregated genetic markers in Salmonella enterica serovar Typhimurium variant 4,[5],12:i:-. Scientific Reports, 8(1), 1–12. `Full Text <https://doi.org/10.1038/s41598-018-33266-5>`_

Radford, D., Strange, P., Lepp, D., Hernandez, M., Rehman, M. A., Diarra, M. S., & Balamurugan, S. (2018). Genomic and proteomic analyses of Salmonella enterica serovar Enteritidis identifying mechanisms of induced de novo tolerance to ceftiofur. Frontiers in Microbiology, 9(SEP), 1–20. `Full Text <https://doi.org/10.3389/fmicb.2018.02123>`_

Rehman, M. A., Carrillo, C., Malouin, F., Diarra, M.S. (2017). Draft whole-genome sequences of multidrug-resistant Escherichia coli O157:H7 strains isolated from feedlot cattle treated with growth-promoting agents. Genome Announcements, 5(18):e00284-17. `Full Text <https://doi.org/10.1128/genomeA.00284-17>`_

Rehman, M. A., Yin, X., Zaheer, R., Goji, N., Amoako, K. K., McAllister, T., & Diarra, M. S. (2018). Genotypes and Phenotypes of Enterococci Isolated From Broiler Chickens. Frontiers in Sustainable Food Systems, 2(December). `Full Text <https://doi.org/10.3389/fsufs.2018.00083>`_

Rehman, M. A., Hasted, T.-L., Persaud-Lachhman, M. G., Yin, X., Carrillo, C., & Diarra, M. S. (2019). Genome Analysis and Multiplex PCR Method for the Molecular Detection of Coresistance to Cephalosporins and Fosfomycin in Salmonella enterica Serovar Heidelberg. Journal of Food Protection, 82, 1938–1949.  `Full Text <https://doi.org/10.4315/0362-028X.JFP-19-205>`_

Robertson, J., Lin, J., Wren-Hedegus, A., Arya, G., Carrillo, C., & Nash, J. H. E. (2019). Development of a multi-locus typing scheme for an Enterobacteriaceae linear plasmid that mediates inter-species transfer of flagella. PLOS ONE, 14, e0218638. `Full Text <https://doi.org/10.1101/664508>`_

Robertson, J., & Nash, J. H. E. (2018). MOB-suite: software tools for clustering, reconstruction and typing of plasmids from draft assemblies. Microbial Genomics, 4(8). `Full Text <https://doi.org/10.1099/mgen.0.000206>`_

Scott, A., Tien, Y.-C., Drury, C. F., Reynolds, W. D., & Topp, E. (2018). Enrichment of antibiotic resistance genes in soil receiving composts derived from swine manure, yard wastes, or food wastes, and evidence for multiyear persistence of swine Clostridium spp. . Canadian Journal of Microbiology, 64(3), 201–208. `Full Text <https://doi.org/10.1139/cjm-2017-0642>`_

Tien, Y. C., Li, B., Zhang, T., Scott, A., Murray, R., Sabourin, L., … Topp, E. (2017). Impact of dairy manure pre-application treatment on manure composition, soil dynamics of antibiotic resistance genes, and abundance of antibiotic-resistance genes on vegetables at harvest. Science of the Total Environment, 581–582, 32–39. `Full Text <https://doi.org/10.1016/j.scitotenv.2016.12.138>`_

Tymensen, L., Zaheer, R., Cook, S. R., Amoako, K. K., Goji, N., Read, R., … McAllister, T. A. (2018). Clonal expansion of environmentally-adapted Escherichia coli contributes to propagation of antibiotic resistance genes in beef cattle feedlots. Science of the Total Environment, 637–638, 657–664. `Full Text <https://doi.org/10.1016/j.scitotenv.2018.05.021>`_

Zaheer, R., Noyes, N., Ortega Polo, R., Cook, S. R., Marinier, E., Van Domselaar, G., … McAllister, T. A. (2018). Impact of sequencing depth on the characterization of the microbiome and resistome. Scientific Reports, 8(1), 1–11. `Full Text <https://doi.org/10.1038/s41598-018-24280-8>`_


Work Package 1
--------------


Work Package 2
--------------


Work Package 3
--------------


Work Package 4
--------------


Work Package 5
--------------

Awosile, B. B., & Smith, B. A. (2017). Risk assessment modelling of fecal shedding caused by extended-spectrum cephalosporin-resistant Escherichia coli transmitted through waste milk fed to dairy pre-weaned calves. Journal of Dairy Science, 100(12), 9667–9673. `Full Text <https://doi.org/10.3168/jds.2017-13196>`_

Collineau, L., Boerlin, P., Carson, C. A., Chapman, B., Fazil, A., Hetman, B., … Smith, B. A. (2019). Integrating Whole-Genome Sequencing Data Into Quantitative Risk Assessment of Foodborne Antimicrobial Resistance: A Review of Opportunities and Challenges. Frontiers in Microbiology, 10(May), 1–18. `Full Text <https://doi.org/10.3389/fmicb.2019.01107>`_

Murphy, C. P., Carson, C., Smith, B. A., Chapman, B., Marrotte, J., McCann, M., … Parmley, E. J. (2018). Factors potentially linked with the occurrence of antimicrobial resistance in selected bacteria from cattle, chickens and pigs: A scoping review of publications for use in modelling of antimicrobial resistance (IAM.AMR Project). Zoonoses and Public Health, 65(8), 957–971. `Full Text <https://doi.org/10.1111/zph.12515>`_

Nekouei, O., Checkley, S., Waldner, C., Smith, B. A., Invik, J., Carson, C., … Gow, S. (2018). Exposure to antimicrobial-resistant Escherichia coli through the consumption of ground beef in Western Canada. International Journal of Food Microbiology, 272(October 2017), 41–48. `Full Text <https://doi.org/10.1016/j.ijfoodmicro.2018.02.022>`_
